.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "yoga"]
 	path = react_juce/yoga
 	url = ../../facebook/yoga.git
+[submodule "react_juce/hermes"]
+	path = react_juce/hermes
+	url = https://github.com/facebook/hermes.git
```

Comment:
Mind editing this gitmodules file to use a relative submodule URL like above? This way if a user clones the project with HTTP, the submodules will follow suit, and likewise for SSH.

Maybe not a big deal since all these submodules are open source anyway, but that's a practice I try to maintain for submods

Reply:
Ah yeah my bad. Does that mean you need to push up a hermes fork to your user account?

Reply:
Nope, it will just make a relative URL against github.com